src/GoogleAdWordsEventForwarder.js

```diff
@@ -259,6 +355,26 @@
             return true;
         }
 
+        function sendGtagConsentDefaults(payload) {
+            try {
+                gtag('consent', 'default', payload);
+            } catch (e) {
+                console.error('gtag is not available to send consent defaults: ', payload, e);
+                return false;
+            }
+            return true;
+        }
+
+        function sendGtagConsentUpdate(payload) {
+            try {
+                gtag('consent', 'update', payload);
+            } catch (e) {
+                console.error('gtag is not available to send consent update: ', payload, e);
+                return false;
+            }
+            return true;
+        }
```

Comment:
Why try catch at all?  why not just do an if check for  `window.gtag`?

Reply:
I think I was trying to just be consistent with what is already in the other methods in this kit.

I think we can refactor this logic as part of: https://go.mparticle.com/work/SQDSDKS-6165